SeaPublicWebsite/Views/EnergyEfficiency/AskForPostcode.cshtml

```diff
@@ -22,7 +22,7 @@
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
         
-        <form action="@Url.Action("AskForPostcode_Post", "EnergyEfficiency", new {reference = Model.Reference})" method="post" novalidate>
+        <form action="@Url.Action("AskForPostcode_Post", "EnergyEfficiency", new {reference = Model.Reference})" method="post">

```

Comment:
Why remove just this one?

Reply:
By mistake, I probably overlooked it while adding the links